### ISSUE TITLE
stackByTable fails to accumulate data for some site-month-products

### ISSUE THREAD

**Issue by vlahm** [NEONScience/NEON-utilities#87]
**Function**
stackByTable (and loadByProduct, which calls it)

**Describe the bug**
Only the sensor positions dataset is properly extracted from the zip files. This error prevents other datasets from properly writing:

Error in data.table::fwrite(stackedDf, paste0(folder, "/stackedFiles/",  : 
  is.list(x) is not TRUE

**To Reproduce**
```
neonUtilities::loadByProduct(dpID='DP1.20288.001', site='ARIK',
    startdate='2017-03', enddate='2017-05', package='expanded',
    check.size=FALSE)
```

(or run zipsByProduct followed by stackByTable)

**Expected behavior**
Downloaded zip files should be extracted and compiled, and resulting datasets should be written to `savepath`.

**System (please complete the following information):**
 - OS: Ubuntu
 - OS Version: 18.04
 - R Version: 3.6.2

**Additional context**
Other sites run successfully:
```
neonUtilities::loadByProduct('DP1.20288.001', 'HOPB', '2018-01', '2018-03',
    package='expanded', check.size=FALSE)
```

**Comment by cklunch** [comment]
@vlahm Thanks for letting us know about this!
I was able to replicate the problem, it's because of an out-of-date file naming on the product-site-months in question. I had thought we updated everything, but apparently there are a few with the old format still hanging around.

I should be able to fix this on GitHub today, and we're planning to submit a new version of neonUtilities to CRAN in the next couple of weeks. I'll keep you posted.
